apyros/metalog.py

```diff
@@ -50,7 +51,7 @@ def __init__( self, filename=None ):
                 self.zf = zipfile.ZipFile(filename)
                 tmp_meta = [info.filename for info in self.zf.infolist() if info.filename.startswith('meta_')]
                 assert len(tmp_meta) == 1, tmp_meta
-                self.lines = self.zf.open(tmp_meta[0]).readlines()
+                self.lines = io.TextIOWrapper(io.BytesIO(self.zf.open(tmp_meta[0]).read())).readlines()  # Py3 binarray
```

Comment:
It took me quite a while to decipher. Is that a result of running 2to3? If I were to do it:
```python
self.lines = self.zf.open(tmp_meta[0]).read().split(b'\n')
```
I am not sure if you want `self.lines` to be unicode or not (above example is bytes).

Reply:
Unicode version if the original data is in ascii:
```python
self.lines = str(self.zf.open(tmp_meta[0]).read(), 'ascii').split('\n')
```

Reply:
Thanks - only the first commit is automatic conversion 2to3. The rest are my hacks/workaround/fixes which I discovered during Python3 usage (and I am afraid that the list of issues is not completed yet). So maybe ignore the first commit and if you have energy to commit fixes after that, like this example above, it would be great. Thanks m.